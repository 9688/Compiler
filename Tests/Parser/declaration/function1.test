int f(){
};

char** f2(){
};

void f3(){
}

void* f4(){
} 