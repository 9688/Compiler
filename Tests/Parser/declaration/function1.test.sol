<<<table #1>>>
<tags>
<types>
int
char
double
void
<vars>
var printf type function printf(var  type  to char, ){
} returned void
var scanf type function scanf(var  type  to char, ){
} returned void
var getchar type function getchar(){
} returned int
var f type function (){
              <<<table #3>>>
              <tags>
              <types>
              <vars>
              юдд{empty block}

} returned int
var f2 type function (){
               <<<table #5>>>
               <tags>
               <types>
               <vars>
               юдд{empty block}

} returned pointer to pointer to char
var f3 type function (){
               <<<table #7>>>
               <tags>
               <types>
               <vars>
               юдд{empty block}

} returned void
var f4 type function (){
               <<<table #9>>>
               <tags>
               <types>
               <vars>
               юдд{empty block}

} returned pointer to void

