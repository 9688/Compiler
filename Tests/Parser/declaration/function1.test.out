<<<table #1>>>
<tags>
<types>
int
char
double
void
<vars>
var f type function (){
              <<<table #3>>>
              <tags>
              <types>
              <vars>
              юдд{empty block}

} returned int
var f2 type function (){
               <<<table #5>>>
               <tags>
               <types>
               <vars>
               юдд{empty block}

} returned pointer to pointer to char
var f3 type function (){
               <<<table #7>>>
               <tags>
               <types>
               <vars>
               юдд{empty block}

} returned void
var f4 type function (){
               <<<table #9>>>
               <tags>
               <types>
               <vars>
               юдд{empty block}

} returned pointer to void

